### PR TITLE
chore(#624): change config parameters to use for redirect url

### PR DESCRIPTION
**Pull request by dipak-pawar**
Fixes: #624

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @dipak-pawar for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by fabric8cd** [comment]
@dipak-pawar snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-625-1`

File: minishift/openshift/auth.app.yaml [hunk 1]
```diff
@@ -75,6 +75,11 @@ objects:
               configMapKeyRef:
                 name: ${SERVICE_NAME}
                 key: notification.serviceurl
+          - name: AUTH_UI_URL
```
Comment (xcoulon) [inline minishift/openshift/auth.app.yaml]: if we really need such a URL configured in `auth` (which I'm not really a fan of), then we'll also have to declare it in https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.app.yaml#L31

File: minishift/openshift/auth.config.yaml [hunk 2]
```diff
@@ -33,3 +33,5 @@ objects:
   data:
     wit.url: http://wit
     notification.serviceurl: http://f8notification
+ #  you can set "ui.url" to your locally deployed ui. by default it's localhost:8080
+ #  ui.url: http://ui
```
Comment (xcoulon) [inline minishift/openshift/auth.config.yaml]: I assume there's a variable with the default value set to `localhost:8080` declared in https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go ?

File: openshift/auth.config.yaml [hunk 3]
```diff
@@ -50,7 +50,7 @@ objects:
     notapproved_redirect: ""
     keycloak.url: https://sso.openshift.io
     notification.serviceurl: ""
-    email.verify.url: https://prod-preview.openshift.io/_home
-    invitation.accepted.url: https://prod-preview.openshift.io/_home
+    email.verify.url: https://openshift.io/_redirects/_verifyEmail
```
Comment (xcoulon) [inline openshift/auth.config.yaml]: AFAIK, this file is just an example. If there's something to fix on prod and pre-prod, we'll have to open a HK to update the configmaps

**Comment by sbose78** [comment]
These should only be the defaults.

**Comment by sbose78** [comment]
@dipak-pawar Should we close this?

**Pull request closed without merging**